README.md

```diff
@@ -56,7 +56,7 @@ For more details, see [The Future of Bandwhich #275](https://github.com/imsnif/b
 
 ### Download a prebuilt binary
 
-If you're on linux, you can download the generic binary from the releases.
+If you're on `android` or `linux``, you can download the generic binary from the [releases](https://github.com/imsnif/bandwhich/releases)
```

Comment:
I think you've got an extraneous backtick here?

Reply:
Done.